### DISCUSSION TITLE
Detected Virus

### DISCUSSION THREAD

**Discussion by ITZDMusic** [ZL-Audio/ZLEqualizer#134] [General]
Hello, i Installed ZL EQ and my Anti Virus detected a Virus. 

Why  Detects a Virus, i dont understand. 
I Deactivate this, and installed, turn it on, bamm Virus Detected.

Have somebody the same problem?

**Comment by zsliu98** [comment]
Similar to #131. Here are the VirusTotal scan results of the 0.4.5 Windows installers:

https://www.virustotal.com/gui/file/cc06f0323efcc407c07fcc6bb1d64137360fba8229d7994c77be1f79490d03d5

https://www.virustotal.com/gui/file/0ce1ffd90c8f405ce607daf9859cdbaf7c685f9eabb2ebcb08cae94bf59ada05

Those two vendors mark EACH version of this plugin as malicious (probably because of the lack of code-signing). Unfortunately, there is not much I can do about that.

If you trust the provided code/binaries from me, you could let the anti-virus software ignore the warning. Otherwise, you may review the code and compile it on your computer.
